src/main/java/com/spring/core/dao/impl/InMemoryBasketDaoImpl.java

```diff
@@ -0,0 +1,64 @@
+package com.spring.core.dao.impl;
+
+import com.spring.core.dao.BasketDao;
+import com.spring.core.model.Product;
+
+import java.util.ArrayList;
+import java.util.List;
+import java.util.Objects;
+
+/**
+ * This class implements BasketDao interface which provides methods to manage products basket
```

Comment:
Here can be provided details about specific realization of this class, I mean that it's a InMemory storage